pkg/apis/extensions/v1alpha1/types_controlplane.go

```diff
@@ -92,7 +94,9 @@ type Purpose string
 
 const (
 	// Normal triggers the ControlPlane controllers for the shoot provider.
+	// Deprecated: This value is no longer necessary and will be removed in a future release.
 	Normal Purpose = "normal"
 	// Exposure triggers the ControlPlane controllers for the exposure settings.
+	// Deprecated: ControlPlane with purpose "exposure" is not deployed since the enablement of SNI.
```

Comment:
```suggestion
	// Exposure triggers the ControlPlane controllers for the exposure settings.
	//
	// Deprecated: ControlPlane with purpose "exposure" is not deployed since the enablement of SNI.
```